src/chain/electrum.rs

```diff
@@ -185,8 +185,13 @@ impl ElectrumRuntimeClient {
 		let txid = tx.compute_txid();
 		let tx_bytes = tx.encode();
 
-		let spawn_fut =
-			self.runtime.spawn_blocking(move || electrum_client.transaction_broadcast(&tx));
+		let spawn_fut = if let Ok(spawn_fut) =
+			self.runtime.spawn_blocking(move || electrum_client.transaction_broadcast(&tx))
+		{
+			spawn_fut
+		} else {
+			return;
```

Comment:
Silently no broadcast?

Not sure if there are other sites where a runtime problem is silently suppressed. I don't think that is desirable?